src/CodeGen_C.cpp

```diff
@@ -25,6 +25,7 @@ const string buffer_t_definition =
     "#ifndef BUFFER_T_DEFINED\n"
     "#define BUFFER_T_DEFINED\n"
     "#include <stdint.h>\n"
+    "#pragma pack(push, 1)\n"
```

Comment:
Note also that the generic target triple is only used to convert parts of the runtime from c++ to bitcode. We specify a real target triple before generating machine code.